src/routes/CommunitySlice.svelte

```diff
@@ -0,0 +1,281 @@
+			quote: '"meds"',
+			class: 'outline-cyan-500'
+		},
+		{
+			image: '/imgs/profile_pictures/outfoxxed.webp',
+			coordinates: [648, 364],
+			size: 80,
+			class: 'outline-orange-500'
+		},
+		{
+			image: '/imgs/profile_pictures/kirottum.webp',
+			coordinates: [772, 651],
+			size: 62,
+			class: 'outline-purple-500'
+		},
+		{
+			image: '/imgs/profile_pictures/beardwarrior.webp',
+			coordinates: [736, 277],
+			size: 87,
+			class: 'outline-amber-500'
+		},
+		{
+			image: '/imgs/profile_pictures/neoney.webp',
+			coordinates: [898, 364],
+			size: 68,
+			class: 'outline-green-500'
+		},
+		{
+			image: amongUsGreenImage,
+			coordinates: [873, 224],
+			size: 79,
+			class: 'outline-green-500'
+		},
+		{
+			image: '/imgs/profile_pictures/SimplyKyle!.webp',
+			coordinates: [859, 159],
+			size: 39,
+			class: 'outline-rose-500'
+		},
+		{
+			image: '/imgs/profile_pictures/sioodmy.webp',
+			coordinates: [974, 107],
+			size: 48,
+			class: 'outline-amber-500'
+		},
+		{
+			image: '/imgs/profile_pictures/flafy.webp',
+			coordinates: [147, 553],
+			size: 87,
+			class: 'outline-pink-500'
+		},
+		{
+			image: '/imgs/profile_pictures/vagahbond.webp',
+			coordinates: [65, 643],
+			size: 74,
+			class: 'outline-amber-500 '
+		},
+		{
+			image: '/imgs/profile_pictures/flick0.webp',
+			coordinates: [263, 653],
+			size: 65,
+			class: 'outline-stone-500 '
+		},
+		{
+			image: '/imgs/profile_pictures/jacekpoz.svg',
+			coordinates: [893, 622],
+			size: 80,
+			class: 'outline-yellow-500 bg-black ',
+			quote: '"piss blob"'
+		},
+		{
+			image: '/imgs/profile_pictures/aylur.webp',
+			coordinates: [354, 798],
+			size: 80,
+			class: 'outline-amber-500 bg-black '
+		},
+		{
+			image: '/imgs/profile_pictures/aleph.nought.webp',
+			coordinates: [583, 824],
+			size: 40,
+			class: 'outline-blue-500 bg-black '
+		}
+	].sort(({ size: a }, { size: b }) => b - a)
+
+	setContext(contextId, {
+		biggestSize: profiles.reduce(
+			(previousSize, { size }) => (size > previousSize ? size : previousSize),
+			1
+		),
+		smallestSize: profiles.reduce(
+			(previousSize, { size }) => (size < previousSize ? size : previousSize),
+			Number.POSITIVE_INFINITY
+		),
+		getSectionElement: () => sectionElement
+	})
+</script>
+
+<section
+	class="relative -mb-[200px] flex h-[1100px] min-h-max w-screen flex-col items-center"
+	bind:this={sectionElement}
+>
+	<Title>
+		<span slot="title">Join a great<br />community</span>
+
+		<span slot="subtitle">
+			Get help from Distro Hoppers, Haiku writers,<br />Hydrohomies and human_(probably)
+		</span>
+	</Title>
+
+	<div class="group mt-16 flex flex-col items-center">
+		<a
+			class="discord p-4"
+			href="https://discord.com/invite/hQ9XvMUjjr"
+			target="_blank"
+			rel="noopener"
+			aria-label="Join us on Discord"
+		>
+			<DiscordIcon class="h-full w-full  " />
+		</a>
+
+		<a href="https://discord.com/invite/hQ9XvMUjjr">
+			<Button type="fancyOutline">Join us on Discord</Button>
+		</a>
```

Comment:
I'm not familiar with Sveltekit, but in Next we usually have routes such as `domain.tld/discord` that we can then use inside the code to reduce redundancy and improve maintainability. If it's possible (and I'm guessing it is), I'll recommend poiting a single `hyprland.org/discord` to the discord invite and using that route in all discord links